unlock-app/src/components/interface/buttons/lock/Withdraw.js

```diff
@@ -1,11 +1,53 @@
 import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+
 import Svg from '../../svg'
 import { LockButton } from '../Button'
+import UnlockPropTypes from '../../../../propTypes'
+import { withdrawFromLock } from '../../../../actions/lock'
+
+export function Withdraw({ lock, withdraw, account, ...props }) {
```

Comment:
Nit: I think you could use a stateless function here and just put the if statement in the onclick?

No strong opinion.